fic_modules/git.py

```diff
@@ -489,33 +505,38 @@ def filter_git_commit_keyword(repository_name, repository_path):
     last_checked = last_check(repository_name)
     new_commit_dict = {"0": {"lastChecked": str(datetime.utcnow())}}
     if limit_checker():
-        new_commits = GIT.get_repo(repository_path).get_commits(since=last_checked)
+        new_commits = GIT \
```

Comment:
Weird break.
Please put new line after `.get_repo(repository_path)`

Reply:
Removed it and searched for similar patterns and also corrected them.